docs/idl.md

```diff
@@ -78,16 +78,18 @@ The available data types for a record, argument, or return value are:
  - Primitives (`i8`, `i16`, `i32`, `i64`, `f32`, `f64`).
  - Strings (`string`)
  - Binary (`binary`). This is implemented as `std::vector<uint8_t>` in C++, `byte[]` in Java,
-   `NSData` in Objective-C, and an object supporting the `buffer` interface in Python.
+   `NSData` in Objective-C, an object supporting the `buffer` interface in Python, and `array<System::Byte>` in C++/CLI..
  - Date (`date`).  This is `chrono::system_clock::time_point` in C++, `Date` in Java,
-   `NSDate` in Objective-C, and `datetime.datetime` in Python.
+   `NSDate` in Objective-C, `datetime.datetime` in Python, and `System::DateTime` in C++/CLI.
  - List (`list<type>`). This is `vector<T>` in C++, `ArrayList` in Java, `NSArray`
-   in Objective-C, and `List` in Python. Primitives in a list will be boxed in Java and Objective-C.
+   in Objective-C, `List` in Python, and `System::Collections::Generic::List` in C++/CLI. 
+   Primitives in a list will be boxed in Java and Objective-C.
  - Set (`set<type>`). This is `unordered_set<T>` in C++, `HashSet` in Java, `NSSet` in
-   Objective-C, and `Set` in Python. Primitives in a set will be boxed in Java and Objective-C.
+   Objective-C, `Set` in Python, and `System::Collections::Generic::HashSet` in C++/CLI. 
+   Primitives in a set will be boxed in Java and Objective-C.
  - Map (`map<typeA, typeB>`). This is `unordered_map<K, V>` in C++, `HashMap` in Java, 
-   `NSDictionary` in Objective-C, and `Dictionary` in Python. Primitives in a map will be boxed in Java
-   and Objective-C.
+   `NSDictionary` in Objective-C, `Dictionary` in Python, and `System::Collections::Generic::Dictionary` in C++/CLI. 
+   Primitives in a map will be boxed in Java and Objective-C.
```

Comment:
I wonder if we shouldn't document these as C# rather than C++/CLI types. Since the resulting DLL can be transparently used in C# code, I believe the focus of documentation should be C# unless we're explicitly talking about the conversion.

Reply:
Good point. All types are now documented for C# instead of C++/CLI.